### PR TITLE
DOC-110, DOC-111, DOC-167: JSDoc fixes and tidying

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
- Fix invalid HTML being generated from JSDoc.
- Address whitespace issues.
- Move classes' details above constructor.
- Remove blank Description section from Global page.
- Increase font size of Parameters and Example headings.
- Increase font size of code examples.
- Add Steam to list of limited commerce versions in WalletScriptingInterface.limitedCommerce property.
- Clarify which line3d overlay properties aren't working.
- Fix Entities > ActionType > "spring" doc. [DOC-110]
- Format some value possibilities in lists. [DOC-161]
- Miscellaneous fixes and typos.

Cases: 
- https://highfidelity.atlassian.net/browse/DOC-111
- https://highfidelity.atlassian.net/browse/DOC-110
- https://highfidelity.atlassian.net/browse/DOC-161

Note: No executable code changes.

[DOC-110]: https://highfidelity.atlassian.net/browse/DOC-110
[DOC-161]: https://highfidelity.atlassian.net/browse/DOC-161

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21421/HighFidelity-Beta-PR16224-5a4ab1d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21421/HighFidelity-Quest-Beta-PR16224-5a4ab1d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21421/HighFidelity-Beta-PR16224-5a4ab1d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21421/HighFidelity-Beta-Interface-PR16224-5a4ab1d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21421/HighFidelity-Beta-PR16224-5a4ab1d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21421/HighFidelity-Beta-Interface-PR16224-5a4ab1d.exe

File: libraries/midi/src/Midi.h [hunk 1]
```diff
@@ -23,7 +23,7 @@
 /**jsdoc
  * The <code>Midi</code> API provides the ability to connect Interface with musical instruments and other external or virtual
```
Comment (jennaingersoll) [inline libraries/midi/src/Midi.h]:
```suggestion
 * <p>The <code>Midi</code> API provides the ability to connect Interface with musical instruments and other external or virtual 
```

File: libraries/midi/src/Midi.h [hunk 2]
```diff
@@ -23,7 +23,7 @@
 /**jsdoc
  * The <code>Midi</code> API provides the ability to connect Interface with musical instruments and other external or virtual 
  * devices via the MIDI protocol. For further information and examples, see the tutorial: 
- * <a href="https://docs.highfidelity.com/en/rc81/script/midi-tutorial.html">Use MIDI to Control Your Environment</a>.</p>
+ * <a href="https://docs.highfidelity.com/en/rc81/script/midi-tutorial.html">Use MIDI to Control Your Environment</a>.
```
Comment (jennaingersoll) [inline libraries/midi/src/Midi.h]:
```suggestion
 * <a href="https://docs.highfidelity.com/script/midi-tutorial.html">Use MIDI to Control Your Environment</a>.
```

Comment (jennaingersoll) [reply libraries/midi/src/Midi.h]: This is the redirect to the latest version of the docs.

File: interface/src/ui/overlays/Overlays.cpp [hunk 3]
```diff
@@ -1916,41 +1915,45 @@ QVector<QUuid> Overlays::findOverlays(const glm::vec3& center, float radius) {
...
  * @property {Vec3} start - The start point of the line. Synonyms: <code>startPoint</code> and <code>p1</code>.
- *     <p><em>If <code>parentID<code> is set, use <code>localStart</code> to set the local position of the start point.</em></p>
- *     <CURRENTLY BROKEN>
+ *     <p><strong>Note:</strong> If <code>parentID</code> is set, use <code>localStart</code> to set the local position of the 
+ *     start point.</p>
  * @property {Vec3} end - The end point of the line. Synonyms: <code>endPoint</code> and <code>p2</code>.
- *     <p><em>If <code>parentID<code> is set, use <code>localEnd</code> to set the local position of the end point.</em></p>
- *     <CURRENTLY BROKEN>
+ *     <p><strong>Note:</strong> If <code>parentID</code> is set, use <code>localEnd</code> to set the local position of the 
+ *     end point.</em></p>
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply interface/src/ui/overlays/Overlays.cpp]: Fixed.

File: libraries/animation/src/AnimInverseKinematics.h [hunk 4]
```diff
@@ -73,12 +73,14 @@ class AnimInverseKinematics : public AnimNode {
...
      *       close to the previous solution, but still provides the benefits of avoiding limb locking.</td></tr>
-     *     <tr><td><code>2</code></td><td>PreviousSolution</td><td>The IK system will begin to solve from the same position and 
-     *       orientations for each joint that was the result from the previous frame.<br />
-     *       Pros: As the end effectors typically do not move much from frame to frame, this is likely to converge quickly 
-     *       to a valid solution.<br />
-     *       Cons: If the previous solution resulted in an awkward or uncomfortable posture, the next frame will also be 
-     *       awkward and uncomfortable. It can also result in locked elbows and knees.</td></tr>
+     *     <tr><td><code>2</code></td><td>PreviousSolution</td><td>
+     *       <p>The IK system will begin to solve from the same position and orientations for each joint that was the result 
+     *       from the previous frame.<p>
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply libraries/animation/src/AnimInverseKinematics.h]: Fixed.

File: libraries/controllers/src/controllers/Actions.cpp [hunk 5]
```diff
@@ -171,7 +171,7 @@ namespace controller {
      *     <tr><td><code>RightFoot</code></td><td>number</td><td>{@link Pose}</td><td>Set the right foot pose of the user's
      *       avatar.</td></tr>
      *
-     *     <tr><td colSpan=4><strong><strong>Application</strong></td>
+     *     <tr><td colSpan=4><strong>Application</strong></td>
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply libraries/controllers/src/controllers/Actions.cpp]: Fixed x 6.

File: libraries/midi/src/Midi.h (hunk 1 above)
Comment (ctrlaltdavid) [reply libraries/midi/src/Midi.h]: Using a <p> for the first paragraph at the start of an API description is optional. Most APIs' JSDoc don't have a <p> here. (Though I could update them to have a <p> if you'd like.)

File: libraries/midi/src/Midi.h (hunk 2 above)
Comment (ctrlaltdavid) [reply libraries/midi/src/Midi.h]: Fixed.

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21784/HighFidelity-Beta-PR16224-1106d8f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21784/HighFidelity-Quest-Beta-PR16224-1106d8f.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21784/HighFidelity-Beta-PR16224-1106d8f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21784/HighFidelity-Beta-Interface-PR16224-1106d8f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21784/HighFidelity-Beta-PR16224-1106d8f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21784/HighFidelity-Beta-Interface-PR16224-1106d8f.exe

File: libraries/midi/src/Midi.h (hunk 1 above)
Comment (jennaingersoll) [reply libraries/midi/src/Midi.h]: your fix works just as well, I just added <p> to match the existing </p>. You removed the </p> works. Thanks!

**Pull request merged**